src/wizard/javascript/nextjs.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Next.js
+doc_link: https://docs.sentry.io/platforms/javascript/guides/nextjs/
+support_level: production
+type: framework
+---
+
+To use Sentry with your Next.js application, you need to use `@sentry/nextjs` (Sentry’s Next.js SDK):
+
+```bash
+yarn add @sentry/nextjs
+# or
+npm install --save @sentry/nextjs
+```
+
+To [configure your app automatically](https://docs.sentry.io/platforms/javascript/guides/nextjs/#configure) (although you can also [configure it manually](https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/)), you can run the sentry-wizard:
+
+```bash
+npx @sentry/wizard -i nextjs
+```
+
+It will automatically patch your application:
+
+- create `sentry.client.config.js` and `sentry.server.config.js` with the default `Sentry.init`.
+- create `next.config.js` with the default configuration.
+- create `sentry.properties` with configuration for sentry-cli (which is used when automatically uploading source maps).
+
+Lastly, to capture [Next.js API route errors](https://nextjs.org/docs/api-routes/introduction), you need to wrap your handlers with a Sentry function:
+
+```javascript
+import { withSentry } from '@sentry/nextjs';
+
+const handler = async (req, res) => {
+  // ...
+}
+
+export default withSentry(handler);
+```
+
+The above configuration has automatic error tracking with source maps for both JavaScript and TypeScript. To reduce the volume of performance data captured, change `tracesSampleRate` to a value between 0 and 1.
```

Comment:
I'd suggest linking to the Sampling content here.  Also, `tracesSampleRate` isn't in the code sample, perhaps I'm missing it? Nonetheless, we typically direct folks to adjust this by commenting the code, not in the prose. 

From the browser wizard, for example:

  // Set tracesSampleRate to 1.0 to capture 100%
  // of transactions for performance monitoring.
  // We recommend adjusting this value in production
  tracesSampleRate: 1.0,

We recommend adjusting the value of `tracesSampleRate` in production. Learn more about configuring sampling in our [full documentation](https://docs.sentry.io/platforms/javascript/configuration/sampling/)."